### ISSUE TITLE
Tooltip for the 'CW' button is not very helpful.

### ISSUE THREAD

**Issue by ryliejamesthomas** [florence-social/mastodon-fork#115]
When you hover over the buttons below the toot composing area in the leftmost column they all have reasonably helpful tooltips:
* Add media ([...])
* Add a poll
* Adjust status privacy

But for the CW button, it just says 'Text is not hidden', and changes to 'Text is hidden behind content warning' when it's toggled.

As far as I can see the lines that would need to be changed are:
* Number 84 in https://github.com/florence-social/mastodon-fork/blob/f6f156fe1bedd83f3b7c9b3997d60d0e52668975/app/javascript/mastodon/locales/en.json (also in a bunch of untranslated locales, if that matters?)
* Number 8 in https://github.com/florence-social/mastodon-fork/blob/f6f156fe1bedd83f3b7c9b3997d60d0e52668975/app/javascript/mastodon/features/compose/containers/spoiler_button_container.js

That's easy enough for me to do a PR for, just want to make sure that's all that needs to be changed.

**Comment by mal0ki** [comment]
Sounds like it'd be useful to improve yes. But let's decide to what before making a PR? 
What would you want to write?

**Comment by ryliejamesthomas** [comment]
Yeah, that's the other part of why I posted. Forgot to say that :|

Was thinking: 'Add a content warning' and 'Content warning included' as the two states?
